### PR TITLE
[TIMOB-14565] iOS: Fix default font-sizes (parity)

### PR DESCRIPTION
**Pull request by hansemannn**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-14565

### CODE REVIEW

File: iphone/Classes/WebFont.m [hunk 1]
```diff
@@ -341,24 +340,16 @@ +(WebFont *)defaultItalicFont
...
 +(WebFont *)fontWithName:(NSString*)name
 {
 	WebFont * result = [[[self alloc] init] autorelease];
 	result.family = [[name copy] autorelease];
-	result.size = 15;
+	result.size = 17;
 	return result;
 }
 
-+(WebFont *)tableRowFont
```
Comment (vijaysingh-axway) [inline iphone/Classes/WebFont.m]: Why deleted ?

Comment (hansemannn) [inline iphone/Classes/WebFont.m]: Unused since iOS 7 was created, just a quick clean-up.

**Review by vijaysingh-axway — APPROVED** [review]
CR and FT passed

**Review by ssjsamir — APPROVED** [review]
**FR Passed** Font sizes now align the defaults to the native defaults.

**Test steps:** 
+ Created a Titanium SDK using this PR
+ Created a new titanium project
+ Copied the code from the test case in https://jira.appcelerator.org/browse/TIMOB-14565
+ Ran the project
+ Pressed `just a test label`
+ Text does no longer change 

**Test Environement**
Iphone 6S(iOS 10.0)
Appcelerator Command-Line Interface, version 6.2.2
Operating System Name: Mac OS X El Capitan
Operating System Version: 10.11.6
Node.js Version: 6.10.1
Xcode: 8.2
Appcelerator Studio: 4.9.0.201705251638

**Pull request merged**